bench/proxygen/optimized/HQFramer.ll

```diff
@@ -579,39 +581,45 @@ invoke.cont26:                                    ; preds = %invoke.cont21
 _ZN5folly8OptionalISt4pairImmEEptEv.exit5.i:      ; preds = %.noexc10
   %5 = load i64, ptr %second.i, align 8, !noalias !13
   %sub.i = sub i64 %sub, %5
-  switch i64 %3, label %invoke.cont34 [
-    i64 1, label %invoke.cont40
-    i64 6, label %invoke.cont40
-    i64 7, label %invoke.cont40
-    i64 8, label %invoke.cont40
-    i64 630, label %invoke.cont40
-    i64 16765559, label %invoke.cont40
-    i64 51, label %invoke.cont40
-    i64 727725890, label %invoke.cont40
-    i64 727725891, label %invoke.cont40
+  switch i64 %3, label %sw.epilog.i [
+    i64 1, label %sw.bb.i
+    i64 6, label %sw.bb.i
+    i64 7, label %sw.bb.i
+    i64 8, label %sw.bb.i
+    i64 630, label %sw.bb.i
+    i64 16765559, label %sw.bb.i
+    i64 51, label %sw.bb.i
+    i64 727725890, label %sw.bb.i
+    i64 727725891, label %sw.bb.i
   ]
 
+sw.bb.i:                                          ; preds = %_ZN5folly8OptionalISt4pairImmEEptEv.exit5.i, %_ZN5folly8OptionalISt4pairImmEEptEv.exit5.i, %_ZN5folly8OptionalISt4pairImmEEptEv.exit5.i, %_ZN5folly8OptionalISt4pairImmEEptEv.exit5.i, %_ZN5folly8OptionalISt4pairImmEEptEv.exit5.i, %_ZN5folly8OptionalISt4pairImmEEptEv.exit5.i, %_ZN5folly8OptionalISt4pairImmEEptEv.exit5.i, %_ZN5folly8OptionalISt4pairImmEEptEv.exit5.i, %_ZN5folly8OptionalISt4pairImmEEptEv.exit5.i
+  %6 = load i64, ptr %settingValue.i, align 8, !noalias !13
+  store i64 %6, ptr %settingValue, align 8, !alias.scope !13
+  br label %invoke.cont34
+
+sw.epilog.i:                                      ; preds = %_ZN5folly8OptionalISt4pairImmEEptEv.exit5.i
+  store i8 0, ptr %settingValue, align 8, !alias.scope !13
+  br label %invoke.cont34
+
 cleanup.thread:                                   ; preds = %.noexc10
   store i64 262, ptr %settingValue, align 8, !alias.scope !13
   call void @llvm.lifetime.end.p0(ptr nonnull %settingValue.i)
   %hasValue.i.i12 = getelementptr inbounds nuw i8, ptr %agg.result, i64 8
   store i64 262, ptr %agg.result, align 8
   store i8 1, ptr %hasValue.i.i12, align 8
-  store i8 0, ptr %which_.i.i.i8.i36, align 8
+  store i8 0, ptr %which_.i.i.i.i, align 8
   br label %cleanup46
 
-invoke.cont34:                                    ; preds = %_ZN5folly8OptionalISt4pairImmEEptEv.exit5.i
-  store i8 0, ptr %settingValue, align 8, !alias.scope !13
-  store i8 0, ptr %hasValue.i.i.i.i.i.i, align 8, !alias.scope !13
+invoke.cont34:                                    ; preds = %sw.epilog.i, %sw.bb.i
+  %.sink = phi i8 [ 0, %sw.epilog.i ], [ 1, %sw.bb.i ]
+  store i8 %.sink, ptr %hasValue.i.i.i.i.i.i, align 8, !alias.scope !13
+  store i8 1, ptr %which_.i.i.i.i, align 8, !alias.scope !13
```

Comment:
This `store` is useless to successor `%cleanup`, although it is a cold path to reach `%cleanup`